app/AMQP/AMQPPublisherServiceProvider.php

```diff
@@ -64,15 +64,14 @@ function (Application $app) {
                     $anyOfSpecification,
                     $messageFactory,
                     function () use ($app) {
-                        // Send messages triggered by CultuurKuur to the CLI queue so the migration does not fill the
-                        // regular queue. Hardcoded for now as a quick fix, should be moved to config later so we can
-                        // easily use this workaround later for other planned migrations.
-                        if ($app['api_client_id'] === 'Nnx3vGNuVVX2KJdtzHhwnx15WlTqGCZ8' || // CultuurKuur prod
-                            $app['api_client_id'] === 'I0ozJs9wF2vHll9pB2TtGxazSsEKwjOw' || // CultuurKuur test
-                            $app['api_client_id'] === 'rLlc90TSFBDRQbZwzP5vm2niDXI1CZev' // CultuurKuur acc
-                        ) {
+                        if ($app->offsetExists('api_client_id') && in_array($app['api_client_id'], $app['amqp.publisher.cli.client_ids'], true)) {
```

Comment:
I see now that the `in_array()` will handle it correctly, regardless if the `api_client_id` is null or not.

However since `$app['api_client_id']` is always defined (and `$app['api_key']` as well), regardless of whether it's `null` or not, I think we can remove these `offsetExists()` checks completely.

Reply:
There was still an issue after all, with the `api_key` possibly being null. So then the `toString()` call on it would crash.